float32 airspeed		
float32 groundspeed
float32 alt
float32 climb		
int16 heading
uint16 throttle
